classes/omnect_rust_modemmanager_deps.bbclass

```diff
@@ -0,0 +1,29 @@
+CARGO_NET_OFFLINE ?= "true"
```

Comment:
not to change, but marker for me: we now do this in 3 classes -> should be refactored